pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a set of users (members and roles), teams, and applications. Which in turn make collaboration easier and access control more streamlined than ever for your applications and devices. Organizations are available for use in all balenaCloud accounts, blending in with existing features and workflows all the while empowering fleet owners to build bigger, better managed fleets.
```

Comment:
@pdcastro We haven't released organizations yet to all users. That's the reason you aren't able to see it at this point from your free account (Whether staging or prod). Alison did mention when organizations will be released the feature will be available to all users and won't be part of paid plans. So that's that. 

I do agree documentation needs to be more low key. That's the reason I added the `What can you do` section below the introduction paragraph where the points are clear with no keywords, no-frills. But, yes I agree I have gone too far with that one. :joy:

Reply:
Oh, I missed the new `What can you do` section, sorry. There is some overlap between my suggestion above and the new section, so it needs to be changed a bit.

Reply:
@pdcastro How's it looking now? With screenshots too.